src/services/AudiusBackend.js

```diff
@@ -1033,13 +1033,15 @@ class AudiusBackend {
         newMetadata.website ||
         newMetadata.donation
       ) {
+        const { data, signature } = await AudiusBackend.signData()
         await fetch(`${IDENTITY_SERVICE}/social_handles`, {
           method: 'POST',
           headers: {
-            'Content-Type': 'application/json'
+            'Content-Type': 'application/json',
+            [AuthHeaders.Message]: data,
+            [AuthHeaders.Signature]: signature
           },
           body: JSON.stringify({
-            handle: newMetadata.handle,
```

Comment:
Was this endpoint updated in identity to take the handle from the requesting user with auth middleware?